src/values.rs

```diff
@@ -702,94 +750,64 @@ pub(crate) mod tests {
+    #[test_case([t(1), t(3), t(2), b("/"), t(5), b("*"), b("-")], -6.5; "basic arithmetic 3")]
     // 3 ++ 4 == 5
-    #[test_case([t(3), t(4), b("++")], value!(5); "hypot")]
+    #[test_case([t(3), t(4), b("++")], 5.0; "hypot")]
     // 5 +-+ 3 == 4
-    #[test_case([t(5), t(3), b("+-+")], value!(4); "hypot sub")]
+    #[test_case([t(5), t(3), b("+-+")], 4.0; "hypot sub")]
     // 3 ^ 4 == 81
-    #[test_case([t(3), t(4), b("^")], value!(81); "pow")]
+    #[test_case([t(3), t(4), b("^")], 81.0; "pow")]
     // (1, 2) + (3, 4) == (4, 6)
-    #[test_case([t(1), t(2), b(COMMA), u(PAREN_UNARY), t(3), t(4), b(COMMA), u(PAREN_UNARY), b("+")], value!(4, 6); "vector addition")]
+    #[test_case([t(1), t(2), b(COMMA), u(PAREN_UNARY), t(3), t(4), b(COMMA), u(PAREN_UNARY), b("+")], Point(4.0, 6.0); "vector addition")]
     // (1, 2) * (3, 4) == 11
-    #[test_case([t(1), t(2), b(COMMA), u(PAREN_UNARY), t(3), t(4), b(COMMA), u(PAREN_UNARY), b("*")], value!(11); "dot product")]
+    #[test_case([t(1), t(2), b(COMMA), u(PAREN_UNARY), t(3), t(4), b(COMMA), u(PAREN_UNARY), b("*")], 11.0; "dot product")]
     // 3 * (1, 2) == (3, 6)
-    #[test_case([t(3), t(1), t(2), b(COMMA), u(PAREN_UNARY), b("*")], value!(3, 6); "scalar product")]
+    #[test_case([t(3), t(1), t(2), b(COMMA), u(PAREN_UNARY), b("*")], Point(3.0, 6.0); "scalar product")]
     // angle (3, 3) == 45
-    #[test_case([t(3), t(3), b(COMMA), u(PAREN_UNARY), u("angle")], value!(45); "angle")]
+    #[test_case([t(3), t(3), b(COMMA), u(PAREN_UNARY), u("angle")], 45.0; "angle")]
     // 3 * - 2 == -6
-    #[test_case([t(3), t(2), u("-"), b("*")], value!(-6); "unary minus")]
+    #[test_case([t(3), t(2), u("-"), b("*")], -6.0; "unary minus")]
     // - 2 * 3 == -6
-    #[test_case([t(2), u("-"), t(3), b("*")], value!(-6); "unary minus 2")]
+    #[test_case([t(2), u("-"), t(3), b("*")], -6.0; "unary minus 2")]
     // -(1, 2) * (3, 4) == -11
-    #[test_case([t(1), t(2), b(COMMA), u(PAREN_UNARY), u("-"), t(3), t(4), b(COMMA), u(PAREN_UNARY), b("*")], value!(-11); "unary minus 3")]
+    #[test_case([t(1), t(2), b(COMMA), u(PAREN_UNARY), u("-"), t(3), t(4), b(COMMA), u(PAREN_UNARY), b("*")], -11.0; "unary minus 3")]
     // cos 90 == 0
-    #[test_case([t(90), u("cos")], value!(0); "unary cos")]
+    #[test_case([t(90), u("cos")], 0.0; "unary cos")]
     // sin 90 == 1
-    #[test_case([t(90), u("sin")], value!(1); "unary sin")]
+    #[test_case([t(90), u("sin")], 1.0; "unary sin")]
     // (1, 2) <> (3, 4) & (1, 4) <> (3, 2) == (2, 3)
     #[test_case([
         t(1), t(2), b(COMMA), u(PAREN_UNARY), t(3), t(4), b(COMMA), u(PAREN_UNARY), b("<>"),
         t(1), t(4), b(COMMA), u(PAREN_UNARY), t(3), t(2), b(COMMA), u(PAREN_UNARY), b("<>"), b("&")
-    ], value!(2, 3); "intersect")]
-    // (0, 0) <> (3, 4) ^^ 5 == (4, -3) -> (7, 1)
-    #[test_case([t(0), t(0), b(COMMA), u(PAREN_UNARY), t(3), t(4), b(COMMA), u(PAREN_UNARY), b("<>"), t(5), b("^^")], value!((4, -3) -> (7, 1)); "offset")]
```

Comment:
This test was previously broken, because `Value::eq_bool` would compare two lines as equal if their `Option<(PointId, PointId)>` were the same. Of course when those were both `None`, they would compare equal, rendering this test trivial.